fs/sync/sync.go

```diff
@@ -219,7 +235,14 @@ func (s *syncCopyMove) pairChecker(in *pipe, out *pipe, wg *sync.WaitGroup) {
 		accounting.Stats.Checking(src.Remote())
 		// Check to see if can store this
 		if src.Storable() {
-			if operations.NeedTransfer(s.ctx, pair.Dst, pair.Src) {
+			NoNeedTransfer := false
+			if fs.Config.CompareDest != "" {
```

Comment:
I think this little block of code is duplicated 3 times, once in operations (with a few more things) and twice in sync.

How about factoring it into a function? Say `CompareOrCopyDest` returning a bool and an error most likely?

This would then let us make `CompareDest` and `CopyDest` private in `operations`

Reply:
will do